src/storage/mod.rs

```diff
@@ -206,6 +207,9 @@ impl Display for Command {
             Command::RawGet { ref ctx, ref key } => {
                 write!(f, "kv::command::rawget {:?} | {:?}", key, ctx)
             }
+            Command::Pause { ref ctx, duration } => {
```

Comment:
You can put a `#[cfg(test)]` to every place `Pause` shows up. But I recommend to use trait instead when we are about to refactor it with future.